wpilibc/src/main/native/include/frc/AnalogInput.h

```diff
@@ -280,12 +279,23 @@ class AnalogInput : public wpi::Sendable,
    */
   void SetSimDevice(HAL_SimDeviceHandle device);
 
-  void InitSendable(wpi::SendableBuilder& builder) override;
-
  private:
   int m_channel;
   hal::Handle<HAL_AnalogInputHandle> m_port;
   int64_t m_accumulatorOffset;
 };
 
 }  // namespace frc
+
+namespace wpi2 {
+
+template<>
+struct Sendable<frc::AnalogInput> {
```

Comment:
As this is a template specialization, is there any chance for multiple implementations for the same type?

Reply:
No.  One implementation per type.  This is the same restriction that Struct and Protobuf have.